ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -132,7 +214,7 @@ module.exports = Marionette.LayoutView.extend({
   handleDefault: function() {
     this.$el.toggleClass(
       'is-default',
-      user.getQuerySettings().isTemplate(this.model)
+      
```

Comment:
✏️ remove extra newline

Reply:
Ended up removing this function since the `is-default` styling is no longer used